makefile

```diff
@@ -229,9 +229,12 @@ notebook: requirements dev-requirements
 	@$(JUPYTER) nbextension enable --py widgetsnbextension
 	@echo "+++ Number of notebooks to execute: $$(find ./docs/source/examples -name "*.ipynb" | wc -l) +++"
 	@echo "+++ Executing notebooks in $(PWD) +++"
-	@find ./docs/source/examples -name "*.ipynb" | \
-	 xargs -P4 -I'{}' $(JUPYTER) nbconvert --execute '{}' \
-	  --to notebook --stdout > /dev/null
+
+	$(JUPYTER) nbconvert --execute $$(find ./docs/source/examples -name "*.ipynb") --to notebook --stdout > /dev/null
```

Comment:
why?

Reply:
weird but I initially thought that this was causing 'notebook check' failure. I have undone it

Reply:
Maybe you are correct. I'm now checking this